### PR TITLE
Add trailing 0 to Happy path tests screencast png's

### PR DESCRIPTION
**Pull request by dmytro-ndp**
### What does this PR do?
It fixes screenshot files naming so that they been sorted normally by name:
```
00000001-Java LS initialization-(11:32:3:701).png
00000002-Java LS initialization-(11:32:6:143).png
00000003-Java LS initialization-(11:32:8:392).png
00000004-Java LS initialization-(11:32:10:640).png
00000005-Java LS initialization-(11:32:12:925).png
00000006-Java LS initialization-(11:32:15:171).png
00000007-Java LS initialization-(11:32:17:422).png
00000008-Java LS initialization-(11:32:19:691).png
00000009-Java LS initialization-(11:32:21:931).png
00000010-Java LS initialization-(11:32:24:176).png
00000011-Java LS initialization-(11:32:26:412).png
00000012-Java LS initialization-(11:32:29:805).png
```
PR is about adding trailing 0 to the file number to normalize sorting.

#### Before
How list of files looked previously https://codeready-workspaces-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/view/che-regular-tests/view/K8S/job/CHE-7-Happy-Path-Tests/1117/artifact/e2e/report/executionScreencast/4-Java%20LS%20initialization/:
Screenshot from 2019-08-20 15-52-52

#### After
Screenshot from 2019-08-20 16-16-08

As you can see, last screenshot file will be always last in the list, so that makes heading file starting from "10000" as redundant as well. 

Also, it makes it simple to create animated gif file - by one command within the the directory with screenshots:
```
ffmpeg -framerate 2 -pattern_type glob -i '*.png' -final_delay 100 out.gif
```

#### Updated "After"
Screenshot from 2019-08-21 20-01-35

### Related issue
https://github.com/eclipse/che/issues/14300

### CODE REVIEW

**Review by Katka92 — APPROVED** [review]
I like this change.

**Comment by Katka92** [comment]
Just for curiosity, is it planned to have that animated gif as a part of the report?

File: e2e/driver/CheReporter.ts [hunk 1]
```diff
@@ -79,17 +79,6 @@ class CheReporter extends mocha.reporters.Spec {
       }
     });
 
-    runner.on('test end', async function (test: mocha.Test) {
```
Comment (Ohrimenko1988) [inline e2e/driver/CheReporter.ts:82]: This changes was implementing for 100% of the application state when test ends. Could you keep this changes? Of course it doesn't mean keeping of "10000" name.

Comment (dmytro-ndp) [reply e2e/driver/CheReporter.ts:82]: Could you, please, clarify for which  reasons we should keep this, taking into account that there is list of screenshots normally ordered now, and CheReporter make screenshot finally itself?

Comment (rhopp) [reply e2e/driver/CheReporter.ts:82]: Hehe ;-) I remember this conversation from the time you made this changes in the first place :-D

**Review by Ohrimenko1988 — APPROVED** [review]

**Comment by dmytro-ndp** [comment]
@Katka92 :
> Just for curiosity, is it planned to have that animated gif as a part of the report?

Now it's about having screencast.mp4 like [this one](https://codeready-workspaces-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/Single-Che-PR-check-E2E-Happy-path-tests-against-k8s-on-codenvy-slave9/177/). 
Making animated gif takes too much time.

**Comment by dmytro-ndp** [comment]
Another advantage of creation of screencast video is decreasing the time for archiving build artifacts, and amount of data to save artifacts/ For example, **executionScreencast.zip** of **108 MiB** produces **screencast.mp4** of **14.5 MiB**

**Comment by dmytro-ndp** [comment]
crw-ci-test

**Pull request merged**